shoop/core/models/_orders.py

```diff
@@ -610,6 +610,9 @@ def get_unshipped_products(self):
     def get_status_display(self):
         return force_text(self.status)
 
+    def get_tracking_codes(self):
+        return [shipment.tracking_code for shipment in self.shipments.all() if shipment.tracking_code]
```

Comment:
This could maybe be ``for shipment in self.shipments.exclude(tracking_code=""))``?

Not sure if that's any better